explorer/src/lib/components/statistics-panel/StatisticsPanel.css

```diff
@@ -0,0 +1,104 @@
+.statistics-panel {
+  display: flex;
+  padding: 1rem 1.375rem;
+  flex-direction: column;
+  column-gap: 0.625rem;
+  row-gap: 0.75rem;
+  border-radius: 1.5rem;
+  background-color: var(--surface-color);
+}
+
+.statistics-panel__statistics {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 1.5rem;
+  overflow: hidden;
+}
+
+.statistics-panel__statistics-item {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.3125rem;
+  position: relative;
+  justify-content: center;
+}
+
+.statistics-panel__statistics-item::after,
+.statistics-panel__statistics-item::before {
+  content: "";
+  position: absolute;
+  background-color: var(--primary-color);
+  z-index: 1;
+}
+
+.statistics-panel__statistics-item::after {
+  inline-size: 100vw;
+  block-size: 1px;
+  inset-inline-start: 0;
+  inset-block-start: calc(calc(1.5rem / 2) * -1);
+}
+
+.statistics-panel__statistics-item::before {
+  inline-size: 1px;
+  block-size: 100%;
+  inset-block-start: 0;
+  inset-inline-start: calc(calc(1.25rem / 2) * -1);
+}
+
+.statistics-panel__statistics-item > div {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
```

Comment:
Are these two needed?

Reply:
I do not know what might happen with different values so I want to be sure that they will never wrap. although the default value for a flex element is `row` is still assign it so that it never changes.